nvim_hunspell/builtin.lua

```diff
@@ -0,0 +1,39 @@
+vim.o.spelloptions = "hunspell"
+vim.o.spelllang = "en"
+vim.o.spell = true
+
+local total_time = os.time()
```

Comment:
Better use `vim.loop.hrtime()` for benchmarking.

Reply:
Will do then!